src/contexts/globalLayout/GlobalNavbar/GalleryNavbar/GalleryLeftContent.tsx

```diff
@@ -33,12 +46,32 @@ export default function GalleryLeftContent({ queryRef }: Props) {
   const rightContent = useMemo(() => {
     if (isMobile) {
       return null;
+    } else if (galleryName && query.userByUsername?.username) {
+      const usernameRoute: Route = {
+        pathname: '/[username]',
+        query: { username: query.userByUsername.username },
+      };
+
+      return (
+        <HStack gap={4} align="center">
+          <Link href={usernameRoute}>
+            <UsernameBreadcrumbLink href={route(usernameRoute)}>
+              {query.userByUsername?.username}
+            </UsernameBreadcrumbLink>
+          </Link>
+          <BreadcrumbText>/ {galleryName}</BreadcrumbText>
```

Comment:
will there be a space before the `/` ?

Reply:
Yeah it gets the space from the gap on the parent